src/components/explore/ProportionExplorerView.svelte

```diff
@@ -66,6 +66,10 @@
   }
 
   function filterResponseData(d, agg, key) {
+    // filter out true/false aggregate results in boolean metrics. See: https://github.com/mozilla/glam/pull/1525#issue-714577595
+    if (d[0].metric_type === 'boolean')
+      return d.filter((di) => di.client_agg_type === '');
```

Comment:
This is great. I'm just curious what the filter by `client_agg_type === ''` is doing exactly?

Reply:
Looking at the [boolean data](https://gist.github.com/Iinh/02327f2c69688a1d23c3b6f86b382956) we get from database I noticed:
- if `client_agg_type` === `true`/`false`: percentile data for true/false aggregation
- if `client_agg_type` === "": data for `always, never, sometimes`

Since we want the latter, I filter by that condition accordingly. Not sure if this is too hacky or not?

Reply:
Nope, I like that and it's helpful to know the different values there. Thanks.

Reply:
I'll add a comment here to make it more clear :)